### PR TITLE
Adding helpers for flags and arg validation

### PR DESCRIPTION
**Pull request by nestor-sk**

### CODE REVIEW

File: cmd/ops/arg_validation_helpers.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package ops
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func exactlyNArgsValidator(n int, errMsg string) cobra.PositionalArgs {
```
Comment (uesteibar) [inline cmd/ops/arg_validation_helpers.go:9]: wdyt of having imperative names? `validateExactArgsCount` and `validateMinimumArgsCount`

**Review by uesteibar — APPROVED** [review]

File: cmd/ops/arg_validation_helpers.go (hunk 1 above)
Comment (omh) [reply cmd/ops/arg_validation_helpers.go:9]: With Go being a imperative language I would think that works well :D

**Review by omh — APPROVED** [review]
🚀

File: cmd/ops/arg_validation_helpers.go (hunk 1 above)
Comment (nestor-sk) [reply cmd/ops/arg_validation_helpers.go:9]: Good point about the name, I neither made it Goish not Cobraish. Looking and the [built-in validators](https://github.com/spf13/cobra/blob/main/user_guide.md#positional-and-custom-arguments) the names should be:

- ExactWithErrorArgs(int, string) or simply ExactArgs
- MinimumNWithErrorArgs(int, string) or simply MinimumNArgs

What do you think is the best? What is clear is that I don't like the name I gave them now ;)

Comment (uesteibar) [reply cmd/ops/arg_validation_helpers.go:9]: I'd say `ExactArgs`, the fact that it's a validator already kinda implies there might be errors

**Pull request merged**